parts/k8s/kubernetesinstalls.sh

```diff
@@ -273,6 +273,11 @@ installKubeletAndKubectl() {
 pullContainerImage() {
     CLI_TOOL=$1
     DOCKER_IMAGE_URL=$2
+    if [ ! -z "${PRIVATE_REGISTRY_SERVER}" ]; then
+        if [[ "$CLI_TOOL" == "docker" ]]; then
+            $CLI_TOOL login -u $SERVICE_PRINCIPAL_CLIENT_ID -p $SERVICE_PRINCIPAL_CLIENT_SECRET $PRIVATE_REGISTRY_SERVER
```

Comment:
Does this in practice not work w/ `img`?

Reply:
Seems like it should:

`https://github.com/genuinetools/img`

Reply:
In any event, if `img login` doesn't work in this context, we'll want to document something like the following:

"Private registry support is for default docker (moby) runtime only. It is not supported for non-docker cluster configuration scenarios, e.g., containerd, clear-containers, kata-containers."

Reply:
Added support for `img` as well.